src/main/kotlin/org/jitsi_modified/impl/neomedia/rtp/sendsidebandwidthestimation/SendSideBandwidthEstimationConfig.kt

```diff
@@ -0,0 +1,178 @@
+    companion object {
+        private const val LEGACY_BASE_NAME =
+            "org.jitsi.impl.neomedia.rtp.sendsidebandwidthestimation.sendsidebandwidthestimation"
+
+        /**
+         * The property that specifies the low-loss threshold
+         * (expressed as a proportion of lost packets) when the loss probability
+         * experiment is *not* active
+         */
+        private val defaultLowLossThresholdProp = property<Double> {
+            readOnce()
+            name("jmt.bwe.send-side.low-loss-threshold")
+            fromConfig(JitsiConfig.newConfig)
+        }
+
+        @JvmStatic
+        fun defaultLowLossThreshold(): Double = defaultLowLossThresholdProp.value
+
+        /*
+         * The property that specifies the high-loss threshold
+         * (expressed as a proportion of lost packets) when the loss probability
+         * experiment is *not* active.
+         */
+        private val defaultHighLossThresholdProp = property<Double> {
+            readOnce()
+            name("jmt.bwe.send-side.high-loss-threshold")
+            fromConfig(JitsiConfig.newConfig)
+        }
+
+        @JvmStatic
+        fun defaultHighLossThreshold(): Double = defaultHighLossThresholdProp.value
+
+        /**
+         * The property that specifies the bitrate threshold (in kbps) when the
+         * loss probability experiment is *not* active.
+         */
+        private val defaultBitrateThresholdProp = property<Bandwidth> {
+            name("jmt.bwe.send-side.bitrate-threshold")
+            readOnce()
+            fromConfig(JitsiConfig.newConfig)
+            retrievedAs<String>() convertedBy(Bandwidth.Companion::fromString)
+        }
+
+        @JvmStatic
+        fun defaultBitrateThresholdKbps(): Int = defaultBitrateThresholdProp.value.kbps.toInt()
+
+        /**
+         * The property that specifies the probability of enabling the
+         * loss-based experiment.
+         */
+        private val lossExperimentProbabilityProp = simple<Double>(
+            readOnce = true,
+            legacyName = "$LEGACY_BASE_NAME.lossExperimentProbability",
+            newName = "jmt.bwe.send-side.loss-experiment.probability"
+        )
+
+        @JvmStatic
+        fun lossExperimentProbability(): Double = lossExperimentProbabilityProp.value
+
+        /**
+         * The property that specifies the low-loss threshold
+         * (expressed as a proportion of lost packets) when the loss probability
+         * experiment is active.
+         *
+         * First, we'll check for an experimental value in the old config file,
+         * then we'll check for an experimental value in the new config file,
+         * then we'll fall back to the default value
+         */
+        private val experimentalLowLossThresholdProp = multiProperty<Double> {
+            legacyProperty {
+                readOnce()
+                name("$LEGACY_BASE_NAME.lowLossThreshold")
+            }
+            newProperty {
```

Comment:
With `loss-experiment.low-loss-threshold` defined in `reference.conf`, is it even possible to fall back to the third one?

Reply:
good point.  i'll have to rethink these.

Reply:
Ah ok, it's not as bad as I thought.  You're right that we won't fall through to the third option, though.  I'll fix this up.

Reply:
I guess having both default to the same value in `reference.conf` isn't that terrible

Reply:
Yeah that's what I ended up doing: default the experimental values to the defaults.  Let me know what you think.